webapp/client/public/css/components.css

```diff
@@ -510,6 +510,14 @@ a.back-link:focus a.back-link:active{
 /* RADIO BUTTONS */
 input[type="radio"] {
     opacity: 0;
+    position: absolute;
```

Comment:
You said that you made it also smaller? I do not see that. Nevertheless, this works because it takes them out of the flow for all the other elements.
I would have expected this to be positioned off screen but it's fine like that.

Reply:
Yes, I was wrong. that was just my first approach and I still had it in mind. :x